wsmaster/che-core-api-user/src/main/java/org/eclipse/che/api/user/server/UserService.java

```diff
@@ -106,7 +104,10 @@ public Response create(@ApiParam("New user")
                                                        UnauthorizedException,
                                                        ConflictException,
                                                        ServerException {
-        userDto.withId(generate("user", ID_LENGTH));
+        if (userDto != null) {
+            //should be generated by userManager
```

Comment:
may be if userDto == null or userDto.getId!= null we should throw exception instead of silently set null?